libs/core/src/lib/info-label/info-label.component.html

```diff
@@ -0,0 +1 @@
+<span class="fd-info-label" [ngClass]="buildComponentCssClass"><ng-content></ng-content></span>
```

Comment:
I wonder why do you go with this approach instead of using Directive? This way you don't give the user the option to put the directive on any HTML element, like `<a>, <span>, <div>`, etc.